packages/api/lambdas/sf-semaphore-down.js

```diff
@@ -57,9 +60,10 @@ const isDecrementEvent = (event) =>
  * @param {Object} event - incoming event from Cloudwatch
  */
 async function handleSemaphoreDecrementTask(event) {
-  if (isDecrementEvent(event)) {
-    const message = getEventMessage(event);
-    const queueName = getQueueName(message);
+  const eventMessage = getEventMessage(event);
+  const parsedEvent = await pullStepFunctionEvent(eventMessage);
```

Comment:
Can we rename `parsedEvent` to `executionMessage`?

Reply:
👍

Reply:
380a681